src/hw10_objects_and_classes/Main.java

```diff
@@ -0,0 +1,25 @@
+package hw10_objects_and_classes;
+
+public class Main {
+    public static void main(String[] args) {
+        Author author_1 = new Author("Лев", "Толстой");
+        Book book_1 = new Book(author_1.getFirstnameAuthor(), "Война и Мир", 1867);
+
+        Author author_2 = new Author("Иван", "Тургенев");
+        Book book_2 = new Book(author_1.getFirstnameAuthor(), "Отцы и дети", 1862);
```

Comment:
```suggestion
        Author author2 = new Author("Иван", "Тургенев");
        Book book2 = new Book(author1.getFirstnameAuthor(), "Отцы и дети", 1862);
```
переменные надо называть в camelCase, а не snake_case